AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/ardc_sofar_nrt.py

```diff
@@ -67,6 +67,7 @@ def process_wave_source_id(source_id, incoming_path=None):
             return
 
         if data is not None:
+            data = data[data['latitude'].notnull()]
```

Comment:
there may be instance when longitude is null  so worth also excluding these values

Reply:
Thanks @bpasquer. Hopefully addressed your comments via https://github.com/aodn/data-services/pull/1208/commits/06969cc4f401474d857736920e02f077d382d9a0